ihatemoney/templates/edit_project.html

```diff
@@ -1,9 +1,17 @@
 {% extends "layout.html" %}
 
 {% block js %}
-    $('#delete-project').click(function ()
-    {
-        $(this).html("<a style='color:red; ' href='{{ url_for('.delete_project') }}' >{{_("you sure?")}}</a>");
+
+    $('#delete-project').click(function(){
+        var link = $(this).find('button');
+        link.click(function(){
+            if ($(this).hasClass("confirm")){
+                return true;
+            }
+            $(this).html("{{_("Are you sure?")}}");
+            $(this).addClass("confirm");
+            return false;
+        });
```

Comment:
The event listener is added every time the form is clicked. Let's simplify this:
```suggestion
    let link = $('#delete-project').find('button');
    let deleteOriginalHTML = link.html();
    link.click(function(){
        if (link.hasClass("confirm")){
            return true;
        }
        link.html("{{_("Are you sure?")}}");
        link.addClass("confirm btn-danger");
        return false;
    });

    $('#delete-project').focusout(function() {
        link.removeClass("confirm btn-danger");
        link.html(deleteOriginalHTML);
    });
```

Reply:
I'm a javascript noob, so I trust you on this ;)